pkg/pillar/zedUpload/awsutil/s3sync.go

```diff
@@ -29,6 +30,7 @@ type UpdateStats struct {
 
 type NotifChan chan UpdateStats
 
+// CustomReader  contains the details of Chunks being downloaded
```

Comment:
Yetus doesn't complain about duplicate spaces? Looks odd in any case

Reply:
yeah, it looks odd will change it.